app/src/main/java/edu/uc/forbesne/shoppingsidekick/ui/main/MainFragment.kt

```diff
@@ -27,4 +23,8 @@ class MainFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
```

Comment:
I modified the order of the contents of the class for the companion object to be last according to the class layout. This was recommended in the Kotlin best practices for efficiency within the layout. (https://kotlinlang.org/docs/coding-conventions.html#class-layout)

Reply:
Duplicated in an earlier code review.